README.md

```diff
@@ -174,3 +169,5 @@ defaults write org.raspberrypi.Imager.plist telemetry -bool NO
 When using the app, press <kbd>CTRL</kbd> + <kbd>ALT</kbd> + <kbd>X</kbd> to reveal the **Advanced options** dialog.
```

Comment:
https://www.raspberrypi.org/blog/raspberry-pi-imager-update-to-v1-6/ says the magic key sequence is ‘Ctrl-Shift-X’ rather than the ‘Ctrl-Alt-X’ written here??

Reply:
>https://www.raspberrypi.org/blog/raspberry-pi-imager-update-to-v1-6/ says the magic key sequence is ‘Ctrl-Shift-X’ rather 
>than the ‘Ctrl-Alt-X’ written here??

Correct.
It's Ctrl-Shift-X

(For some strange reason, those of us who where kids/teens in the 90s may remember ctrl-alt-x better though :-) )

Reply:
Hmm. It's CTRL+SHIFT+X on Windows certainly. `qml/main.qml` says this:

```
Shortcut {
        sequences: ["Shift+Ctrl+X", "Shift+Meta+X"]
        context: Qt.ApplicationShortcut
        onActivated: {
            optionspopup.openPopup()
        }
    }
```

Which I think means SHIFT+CTRL+X **and** SHIFT+<whatever meta is defined as>+X. So I will change the text so it reads CTRL+SHIFT+X.

Reply:
> So I will change the text so it reads CTRL+SHIFT+X.

Done.

Reply:
>Which I think means SHIFT+CTRL+X and SHIFT+whatever meta is defined as+X

Meta is included as alternative key combination because that is "Control" on Mac OS X.